flink/v1.12/flink/src/test/java/com/netease/arctic/flink/FlinkTestBase.java

```diff
@@ -105,66 +102,38 @@ public class FlinkTestBase extends TableTestBase {
   protected Map<String, String> props;
   private volatile StreamExecutionEnvironment env = null;
 
+  public static final Schema TABLE_SCHEMA = new Schema(
```

Comment:
I think this TABLE_SCHEMA is duplicated with BasicTableTestHelper.TABLE_SCHEMA, maybe you can fetch the schema by `BasicTableTestHelper.tableSchema()`